pkg/build/types/parser/deffile.go

```diff
@@ -316,10 +316,11 @@ func populateDefinition(sections map[string]*types.Script, files *[]types.Files,
 	}
 	d.BuildData.Files = *files
 	d.BuildData.Scripts = types.Scripts{
-		Pre:   *sections["pre"],
-		Setup: *sections["setup"],
-		Post:  *sections["post"],
-		Test:  *sections["test"],
+		Default: *sections["default"],
```

Comment:
`default` section is a really generic name, could this be renamed to `arguments` ?

Reply:
reflected in this commit
https://github.com/apptainer/apptainer/pull/1356/commits/40513e1ddb9fb0e7a088272640198fa26eb6b3a1